dao.py

```diff
@@ -619,6 +620,25 @@ def get_user_by_session_id_or_none(self, session_id):
     def get_user_by_region(self, regions):
         pass
 
+    def get_is_superadmin(self, user_id):
+        user = None
+        if self.users_col.find_one({'_id': user_id}):
+            user = M.User.load(self.users_col.find_one({'_id': user_id}))
+            return user.admin_level == 'SUPER'
+        else:
+            return False
+
+    def set_user_superadmin(self, user_id, admin_level):
```

Comment:
this can be used for any admin_level, so let's call it something like `set_user_admin_level` or `update_user_admin_level`? (it might also make more sense to just make a general `update_user` function in the same vein as `update_player` and `update_tournament`).